tests/ops/op_math/test_sum.py

```diff
@@ -85,7 +85,7 @@ def _get_pw(w, pauli_op):
 
 
 # pylint: disable=unused-argument
-def sum_using_dunder_method(*summands, do_queue=False, id=None):
+def sum_using_dunder_method(*summands, id=None):
```

Comment:
I'm not entirely sure, but aren't you calling this function with `do_queue=None` further below? :thinking:

Reply:
Oh nice, catch. I hadn't gone through all the tests yet. Removed the argument in the function call: 20d4ab5cda0501ccefc193e90e263d62f0ff55bd